Unit.py

```diff
@@ -74,8 +81,12 @@ def underattacked(self, damage, underattack_type):
             Unit.alive(self)
 
     def force_attack(self, enemy):
-        getAttack = Attack(self.attack, self.attack_type, self.name)
-        getAttack.sendAttack(enemy)
+        if enemy.position == 'air':
+            getAttack = Attack(self.air_attack, self.air_attack_type, self.name)
+            getAttack.sendAttack(enemy)
+        else:
+            getAttack = Attack(self.attack, self.attack_type, self.name)
```

Comment:
self.attack could rename to self.ground_attack to distinguish between each other